extra/docker/develop/Dockerfile

```diff
@@ -51,20 +50,17 @@ RUN sudo apt-get install -y \
 	socat \
 	sshpass \
 	vim \
-	zsh
-
+	zsh \
 # Misc useful things when developing
-RUN sudo apt-get install -y  \
 	curl \
```

Comment:
Does this work with the comment between escaped newlines?

Reply:
Yes.